gravitee-am-identityprovider/gravitee-am-identityprovider-google/src/main/java/io/gravitee/am/identityprovider/google/GoogleIdentityProviderConfiguration.java

```diff
@@ -88,6 +89,14 @@ public void setConnectTimeout(Integer connectTimeout) {
         this.connectTimeout = connectTimeout;
     }
 
+    public Integer getIdleTimeout() {
```

Comment:
The idleTimeout is never set to the WebClientOptions

https://github.com/gravitee-io/gravitee-access-management/blob/issues%2F%233505-http-request-timeout-social-idp/gravitee-am-identityprovider/gravitee-am-identityprovider-google/src/main/java/io/gravitee/am/identityprovider/google/authentication/spring/GoogleAuthenticationProviderConfiguration.java#L55

Reply:
Changes applied, thanks.

Reply:
Done